website/content/docs/connect/config-entries/mesh.mdx

```diff
@@ -97,6 +98,8 @@ spec:
 </Tab>
 </Tabs>
 
+Note that the Kuberetes example does not include a `partition` field. Configuration entries are applied on Kubernetes using [custom resource definitions (CRD)](/docs/k8s/crds), which can only reference their own partition.
```

Comment:
Your wording seems reasonable for this page, but I don't think it generalizes to the other config entries The partition field in this metadata block is about the scope of the CRD. There can be references to other partitions in the spec block (see your edits to the ingress-gateway config entry docs as an example [here](https://github.com/hashicorp/consul/blame/docs/service-mesh-config-entries-add-partitions--1.11.0/website/content/docs/connect/config-entries/ingress-gateway.mdx#L503)).

```suggestion
Note that the Kuberetes example does not include a `partition` field. Configuration entries are applied on Kubernetes using [custom resource definitions (CRD)](/docs/k8s/crds), which can only be scoped to their own partition.
```

Reply:
@freddygv  - Thanks, but I'm still unclear as to how to explain why the partition param appears in the HCL and JSON, but not in the YAML for other configuration entries. Do you mean that the partition param for other CRDs is specified in the spec block? I want to be clear about the differences between the formats for all configuration entries.